examples/exotica_examples/package.xml

```diff
@@ -17,5 +17,6 @@
   <depend>ik_solver</depend>
   <depend>task_map</depend>
   <depend>robot_state_publisher</depend>
+  <depend>joy</depend>
```

Comment:
Consider making ``run_depend``.

Reply:
We use format 2 for `package.xml` so need `exec_depend` rather than `run_depend`.

Reply:
Yes, I realised that later - sorry, please use ``exec_depend``.